tests/unit/dogstatsd/test_statsd.py

```diff
@@ -313,7 +313,9 @@ def test_timed_context(self):
         Measure the distribution of a context's run time.
         """
         # In seconds
-        with self.statsd.timed('timed_context.test'):
+        with self.statsd.timed('timed_context.test') as timer:
+            self.assertIsInstance(timer,
```

Comment:
I need to update the tests to inherit from `unittest`. In the meantime would you mind using 
```python
t.assert_is_instance(...)
```

rather than 
```python
self.assertIsIstance(...)
```